splunk_connect_for_snmp/common/schema_migration.py

```diff
@@ -67,10 +67,29 @@ def migrate_database(mongo_client, periodic_obj):
 
         for x in range(previous_schema_version, CURRENT_SCHEMA_VERSION):
             fun_name = "migrate_to_version_" + str(x + 1)
-            getattr(sys.modules[__name__], fun_name)(mongo_client, periodic_obj)
+            if str(x + 1) == "4":
+                getattr(sys.modules[__name__], fun_name)(
+                    mongo_client, periodic_obj, chain_of_tasks_expiry_time
+                )
+            else:
+                getattr(sys.modules[__name__], fun_name)(mongo_client, periodic_obj)
 
         save_schema_version(mongo_client, CURRENT_SCHEMA_VERSION)
 
+    previous_expiry_time = periodic_obj.get_chain_of_task_expiry()
+    logger.info(
+        f"Chain of tasks expiry time. Previous: {previous_expiry_time}. Current: {chain_of_tasks_expiry_time}"
+    )
+    expiry_time_changed = False
+    if (
+        previous_expiry_time is None
```

Comment:
why it is done as when previous expiry time is None? that indicates there's no tasks yet